cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -323,6 +325,40 @@ public MetaInformation metaInformation() {
         return replicator;
     }
 
+    /**
+     * Lists replication jobs. Includes replications created via /_replicate endpoint as well as
+     * those created from replication documents. Does not include replications which have
+     * completed or have failed to start because replication documents were malformed. Each job
+     * description will include source and target information, replication id, a history of
+     * recent event, and a few other things.
```

Comment:
event**s**

`and other information`?

Reply:
@ricellis I've copied and pasted from the CouchDB docs where possible/applicable but I agree that this is not the best wording.